### PR TITLE
MEN-4895 Monitoring: RBAC aware alert test scenario

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please-please

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

File: tests/tests/test_monitor_client.py [hunk 1]
```diff
@@ -24,13 +24,16 @@
...
 
+
 from ..MenderAPI import (
     authentication,
     get_container_manager,
     DeviceAuthV2,
     logger,
 )
 
+import testutils.api.useradm as useradm
```
Comment (tranchitella) [inline tests/tests/test_monitor_client.py:35]:
```suggestion
from testutils.api import useradm
```

File: tests/tests/test_monitor_client.py [hunk 2]
```diff
@@ -24,13 +24,16 @@
 from email.policy import default
 from ..common_setup import monitor_commercial_setup_no_client
 
+
```
Comment (tranchitella) [inline tests/tests/test_monitor_client.py:27]: Don't know why you need this new line :)

**Review by tranchitella — APPROVED** [review]
It looks okay to me!

File: tests/tests/test_monitor_client.py (hunk 2 above)
Comment (merlin-northern) [reply tests/tests/test_monitor_client.py]: I have really no idea. must be a symptom of some deeper need for blank lines. removed.

File: tests/tests/test_monitor_client.py (hunk 1 above)
Comment (merlin-northern) [reply tests/tests/test_monitor_client.py]: thanks fixed.

**Comment by merlin-northern** [comment]
pipelies passed before the cosmetic changes: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/348973953

**Pull request merged**